automated install/uninstall.sh

```diff
@@ -206,11 +206,7 @@ removeNoPurge() {
 }
 
 ######### SCRIPT ###########
-if command -v vcgencmd &> /dev/null; then
-    echo -e "  ${INFO} All dependencies are safe to remove on Raspbian"
-else
-    echo -e "  ${INFO} Be sure to confirm if any dependencies should not be removed"
-fi
+echo -e "  ${INFO} Be sure to confirm if any dependencies should not be removed"
```

Comment:
We can probably sort this in another PR, but I've never actually read this wording before.

The sentence doesn't read quite right to me...

Reply:
As the default is currently loop through packages for removal, I guess this is aimed to be a reminder to type "n" they should be kept. But indeed this seems a bit obsolete or else fits better after "y" was chosen to loop through each package. We can discuss in #3929 how to proceed with this topic in general.

Reply:
It's actually more the combination of words made to form an English sentence. Something feels wrong when I say it out loud - but I haven't yet managed to craft a better sentence